src/Microsoft.Extensions.Configuration.Abstractions/IConfigurationBuilder.cs

```diff
@@ -26,11 +26,11 @@ public interface IConfigurationBuilder
         /// </summary>
         /// <param name="source">The configuration source to add.</param>
         /// <returns>The same <see cref="IConfigurationBuilder"/>.</returns>
-        IConfigurationBuilder Add(IConfigurationSource source);
+        new IConfigurationBuilder Add(IConfigurationSource source);
```

Comment:
Is this useful being fluent? Most of the extension methods built on top of this are fluent themselves.